contrib/ci/generate_docker.py

```diff
@@ -78,6 +80,15 @@ def get_container_cmd():
                 wfd.write('RUN dpkg --add-architecture %s\n' % SUBOS)
         elif line == "%%%OS%%%\n":
             wfd.write("ENV OS %s\n" % TARGET)
+        elif "%%%QUBES%%%" in line:
+            if 'qubes' in TARGET:
+                cmd_line = line.replace(
+                    "%%%QUBES%%%",
+                    ', "-Dqubes_enable=true"'
```

Comment:
make sure you update this to `-Dqubes=true` when you fix meson options.

Reply:
Done.